device-registry/src/main/scala/org/genivi/sota/device_registry/DevicesResource.scala

```diff
@@ -201,9 +201,9 @@ class DevicesResource(namespaceExtractor: Directive1[AuthedNamespaceScope],
    * @return      Route object containing routes for creating, deleting, and listing devices
    * @throws      Errors.MissingDevice if device doesn't exist
    */
-  def route: Route = api ~ mydeviceRoutes ~ devicePackagesRoutes
+  def route: Route =
     (pathPrefix("mydevice") & put & path("packages")) {
       updateInstalledSoftwareForDevices()
-    }
+    } ~ api ~ mydeviceRoutes ~ devicePackagesRoutes
```

Comment:
because so much weird stuff has happened with akka http that I've become suspicious. Cargo cult programming.